.gitignore

```diff
@@ -5,10 +5,14 @@ lib-cov
 *.out
 *.pid
 *.gz
+.idea
+.DS_Store
```

Comment:
Technically this is all of the work that Greg did. I believe Phill and I already reviewed this PR once. Have a look anyway!